README.md

```diff
@@ -39,14 +39,17 @@ Full installation documentation can also be found
 
 Get the latest release via pip as
 ```shell
-python -m pip install yt_astro_analysis
+python -m pip install yt-astro-analysis
 ```
 
 Or with conda, as
 ```shell
-conda install -c conda-forge yt_astro_analysis
+conda install -c conda-forge yt-astro-analysis
 ```
 
+Note, the package name is spelled with hyphens (`yt-astro-analysis`)
+when installing from pip or conda.
```

Comment:
I agree that it's best to use hyphens everywhere in the docs but this sentence is slightly wrong since with pip, underscores also work as they are automatically normalised